build-fail-reminder.py

```diff
@@ -188,7 +188,7 @@ def main(args):
                 # Package failed to build for 6 weeks - file a delete request
                 r = osc.core.Request()
                 r.add_action('delete', tgt_project=project, tgt_package=package)
-                r.description = cgi.escape("[botdel] Package has failed to build for >= 6 weeks")
+                r.description = escape("[botdel] Package has failed to build for >= 6 weeks")
```

Comment:
just using `&gt;= 6 weeks` is out of style? :)

Reply:
sure; is an option - I don't mind changing it and not using escape (was just a 1 for 1 replacement without brain muscle)